SECP-service/src/main/resources/assets/app/js/app.js

```diff
@@ -38,6 +38,11 @@ angular.module('SECP', ['ngResource', 'ngRoute', 'ui.bootstrap', 'ui.date',
         controller: 'LoginController',
         css: 'css/login.css',
       })
+      .when('/authenticate', {
```

Comment:
if you are adding a new route. Please update the SECPService.java. @mohameddahrouj  can help you with this.